jira/resources.py

```diff
@@ -899,7 +900,13 @@ def delete(self, moveFixIssuesTo=None, moveAffectedIssuesTo=None):
         return super(Version, self).delete(params)
 
     def update(self, **args):
```

Comment:
it bothers me a little that this is `**args`, we would expect it to follow the convention of `**kwargs`. 

```suggestion
    def update(self, **kwargs):
```

This change would also require changes in the function to how it is used.

Reply:
Well that would be a change in the public API which we should think twice to change...
I wouldn't do a change like this in this pr.

Maybe we should list up small improvements which might break the API and prepare for them

Reply:
Changing this variable name would not affect the public API. 

`**` indicates an unspecified number of **keyword** arguments. This variable is a catch all for these keyword arguments and it's name won't affect users in anyway.

Reply:
darnit indeed you're right `*args ` and `**kwargs`

Reply:
🤔 this change is not tested. I'll add a test for it too